docs/first-steps/management-account.md

```diff
@@ -0,0 +1,152 @@
+# Orchestrate the Management account
+
+Finally we reach the point in which you'll get to actually create the infrastructure in our AWS environment.
+
+Some accounts and layers rely on other accounts/layers being already deployed, creating dependencies between each other and establishing an order in which all layers should be deployed. We will go through these dependency chains in order.
+
+!!! success "Basic Landing Zone AWS Expenses"
+    By default this AWS Reference Architecture configuration should not incur in any costs.
+
+The **management** account is used to configure and access all AWS Organizations managed accounts, also, billing and financial decisions are enforced though this account.
+
+## Deploy the Management account's layers
+
+To begin, place yourself in the `management` account directory.
+
+``` bash
+cd management
+```
+
+### Terraform backend layer
+
+Move into the `base-tf-backend` directory and run:
+
+``` bash
+leverage terraform init
+leverage terraform apply
+```
+
+All `apply` commands will prompt for confirmation, answer `yes` when this happens.
+
+!!! info "More information on [`terraform init`](../user-guide/base-workflow/leverage-cli/reference/terraform#init) and [`terraform apply`](../user-guide/base-workflow/leverage-cli/reference/terraform#apply)"
+!!! note
+    The `apply` command may result in an error similar to:
+
+    ```
+    Error: error creating public access block policy for S3 bucket (ex-security-terraform-backend): 
+    OperationAborted: A conflicting conditional operation is currently in progress against this resource. 
+    Please try again.
+    ```
+
+    If this happens, please re-run the command.
```

Comment:
I think @lgallard recently crafted a fix for this issue here: https://github.com/binbashar/terraform-aws-tfstate-backend/pull/14

Reply:
Great! If that's the case I'll remove the warnings, it also looks better that way 😃

Reply:
@diego-ojeda-binbash Right! I should be fixed in the latest tf-backend module PR 💪🏼 
@angelofenoglio If possible let's test it, and if everything works fine let's take this consideration out of the First Steps doc :)

Reply:
I removed the warnings before reading @exequielrafaela  comment, so I'll revert that commit and then we can simply make a super small PR for that once the changes on the module have been fully tested. Does that sound fair @diego-ojeda-binbash ?